### PR TITLE
Solr bug cleanup

### PR DESCRIPTION
**Pull request by Primer42**
There were a few issue that I found when I quickly looked at the solr code.
Firstly, it is not disabled by default as the docs imply.  In fact, it's always enabled.
Secondly, the default value for `connectionTimeout` was way too low, so I increased it to the default value recommended on the solr wiki.
Thirdly, there was a debug print that someone committed.
Once this is approved and merged, I'll update the default value in the docs accordingly.
@byxorna @maddalab

### CODE REVIEW

**Comment by maddalab** [comment]
lgtm, requires a gh-pages update

File: app/collins/solr/SolrConfig.scala [hunk 1]
```diff
@@ -16,17 +16,18 @@ object SolrConfig extends Configurable {
       messageWithDefault("invalidUrl", "solr.externalUrl %s is invalid".format(t), t)
   }
 
+  # defaults taken from http://wiki.apache.org/solr/Solrj#Changing_other_Connection_Settings
   def embeddedSolrHome = getString("embeddedSolrHome", "NONE")
-  def enabled = true
+  def enabled = getBoolean("enabled", false)
```
Comment (KirilAngov) [inline app/collins/solr/SolrConfig.scala]: Wouldn't we consider this a BC breaking change? Is the `true` defined somewhere else as the default?

Comment (byxorna) [inline app/collins/solr/SolrConfig.scala]: The solr plugin can not ever be disabled, so forcing this to true is ok (collins no worky without solr)

**Comment by byxorna** [comment]
LGTM

File: app/collins/solr/SolrConfig.scala (hunk 1 above)
Comment (KirilAngov) [inline app/collins/solr/SolrConfig.scala]: If @byxorna is right, we need to default it to true then: `def enabled = getBoolean("enabled", true)`

Comment (maddalab) [inline app/collins/solr/SolrConfig.scala]: I'm with @KirilAngov  on this, we should update the docs to say it is enabled by default.

File: app/collins/solr/SolrConfig.scala [hunk 2]
```diff
@@ -16,17 +16,18 @@ object SolrConfig extends Configurable {
       messageWithDefault("invalidUrl", "solr.externalUrl %s is invalid".format(t), t)
   }
 
+  # defaults taken from http://wiki.apache.org/solr/Solrj#Changing_other_Connection_Settings
```
Comment (maddalab) [inline app/collins/solr/SolrConfig.scala]: # is not used for comments in scala

**Comment by maddalab** [comment]
@Primer42 before we merge this we need updates to the reference config so that we do not forget

**Comment by KirilAngov** [comment]
:+1:

**Pull request merged**